File: 1/1-12.hs
pascal :: Int -> Int -> Int
pascal _ 0 = 1                 -- Base case: leftmost element is always 1
pascal row col
    | col == row = 1           -- Base case: rightmost element is always 1
    | otherwise = pascal (row - 1) (col - 1) + pascal (row - 1) col   -- Recursive case

File: 1/1-12.cpp
int pascal(int row, int col) {
    // Base cases: leftmost and rightmost elements are always 1
    if (col == 0 || col == row) {
        return 1;
    }
    // Recursive case: sum of two elements from previous row
    return pascal(row - 1, col - 1) + pascal(row - 1, col);
}